article/app/views/fragments/articleBody.scala.html

```diff
@@ -23,12 +24,14 @@
         class="@RenderClasses(Map(
             "has-feature-showcase-element" -> (article.tags.isFeature && article.elements.hasShowcaseMainElement),
             "paid-content" -> isPaidContent,
-            "paid-content--white" -> (isPaidContent && ActiveExperiments.isParticipating(CommercialPaidContentTemplate))
+            "paid-content--white" -> (isPaidContent && ActiveExperiments.isParticipating(CommercialPaidContentTemplate)),
+            (s"content--pillar-${article.metadata.pillar.nameOrDefault}", experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            (s"content--type-${article.metadata.designType.nameOrDefault}", experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            ("tonal", !experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            (s"tonal--${toneClass(article)}", !experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
```

Comment:
why stop using the arrow notation `key -> value` and introduce the tuple notation `(key, value)` in this `Map`

Reply:
We should definitely stick to a consistent syntax. I would probably stick to using `key -> value` syntax here.

Reply:
no good reason - bad copy/paste. will fix